docs/src/app/components/pages/components/table.jsx

```diff
@@ -494,6 +495,10 @@ class TablePage extends React.Component {
   _onRowSelection(rows) {
     console.log(rows);
   }
+
+  _onRowClick(rows) {
+    console.log(rows)
```

Comment:
I thought it was instructive to point out in the docs that this needed to be attached to <Table /> and not <TableRow />. Intuitively that's where I wanted to attach it, but the component is written differently. What do you think?